htsinfer/extract_random_seq_subset.py

```diff
@@ -0,0 +1,324 @@
+"""Extract a random subset of the sample data."""
+
+import shutil
```

Comment:
FYI: There's some suggested way of how imports should be ordered:

imports from builtin packages
<empty line>
imports from third party packages
<empty line>
imports from internal modules (i.e., of htsinfer package)